src/DIRAC/Core/Utilities/ElasticSearchDB.py

```diff
@@ -390,7 +391,7 @@ def index(self, indexName, body=None, docID=None):
             return S_ERROR("Missing index or body")
 
         try:
-            res = self.client.index(index=indexName, doc_type="_doc", body=body, id=docID)
+            res = self.client.index(index=indexName, doc_type="_doc", body=body, id=docID, op_type=op_type)
```

Comment:
Given
https://github.com/elastic/elasticsearch-py/blob/c3a564ba3e396b3de874e6d82669ffb0a005791a/elasticsearch/client/__init__.py#L361
```python
    def index(self, index, body, doc_type=None, id=None, params=None, headers=None):
```
Shouldn't this be

```python
res = self.client.index(index=indexName, doc_type="_doc", body=body, id=docID, 
                                  params={'op_type': op_type})
```

Reply:
Oh yes, duh. Changed in recent commit .